<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>App</class>
 <widget class="QWidget" name="App">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Music Player</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="search_button">
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="other_menu" native="true">
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
	background-color: rgb(0, 85, 255);
}</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="search_bar">
     <property name="placeholderText">
      <string>Search songs here....</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QWidget" name="playback_control" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
	background-color: rgb(0, 0, 0);
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="main_menu" native="true"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
